python/paddle/fluid/dygraph/dygraph_to_static/loop_transformer.py

```diff
@@ -221,8 +228,8 @@ def visit_Name(self, node):
             self.in_loop_vars[loop_node].add(node)
             if type(node.ctx) in write_context:
                 self.write_in_loop[loop_node].add(node)
-            if self.in_condition:
-                self.condition_vars[loop_node].add(node)
+        if self.in_condition:
```

Comment:
Why un-indent these two lines?

Reply:
Because `condition_vars[loop_node]`  should only contain vars in current `loop_node`, should not contain the vars of **child** loop node or **parent** loop node.

Reply:
OK, thanks for explanation.